src/parser/jobs/whm/modules/Thinair.tsx

```diff
@@ -0,0 +1,143 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import Rotation from 'components/ui/Rotation'
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import React, {Fragment, ReactNode} from 'react'
+import {Accordion} from 'semantic-ui-react'
+
+type ThinAirRecord = {
+	start: number,
+	end: number,
+	casts: number[],
+	mpsaved: number,
+}
+
+export default class Thinair extends Analyser {
```

Comment:
nit: We prefer new modules use named exports instead of a default export.  This would also require changing the import statement in modules/index.js below.

```suggestion
export class Thinair extends Analyser {
```